### PR TITLE
Trm 23600 search fields config - config rest api change

### PR DESCRIPTION
**Pull request by sahmad-ebi**

### CODE REVIEW

File: support-data-rest/pom.xml [hunk 1]
```diff
@@ -184,32 +195,32 @@
...
-									<goal>oas-generate</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+	<!-- <profiles> -->
```
Comment (eddturner) [inline support-data-rest/pom.xml]: should we just delete these commented out lines?

File: support-data-rest/src/main/java/org/uniprot/api/configure/service/UniProtConfigureService.java [hunk 2]
```diff
@@ -34,8 +34,9 @@
         return SEARCH_ITEMS.getSearchItems();
     }
 
-    public List<org.uniprot.store.search.domain2.SearchItem> getUniProtSearchItems2() {
-        return UniProtKBSearchItems.INSTANCE.getSearchItems();
+    public List<AdvanceSearchTerm> getUniProtSearchItems2() {
+        //        return UniProtKBSearchItems.INSTANCE.getSearchItems();
```
Comment (eddturner) [inline support-data-rest/src/main/java/org/uniprot/api/configure/service/UniProtConfigureService.java]: delete

File: support-data-rest/src/main/java/org/uniprot/api/configure/service/UniProtConfigureService.java [hunk 3]
```diff
@@ -34,8 +34,9 @@
...
+        return getUniprotSearchTerms();
```
Comment (eddturner) [inline support-data-rest/src/main/java/org/uniprot/api/configure/service/UniProtConfigureService.java]: Uniprot -> UniProt

File: uniprotkb-rest/pom.xml [hunk 4]
```diff
@@ -265,35 +265,35 @@
...
-									<goal>oas-generate</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+	<!-- <profiles> -->
```
Comment (eddturner) [inline uniprotkb-rest/pom.xml]: delete commented out lines

File: support-data-rest/pom.xml (hunk 1 above)
Comment (sahmad-ebi) [reply support-data-rest/pom.xml]: I have enabled the commented code.

File: uniprotkb-rest/pom.xml (hunk 4 above)
Comment (sahmad-ebi) [reply uniprotkb-rest/pom.xml]: I have enabled the commented code.

**Pull request merged**